VRDR/IJEMortality.cs

```diff
@@ -1703,7 +1703,8 @@ public string DETHNIC4
                     return "N";
                 }
                 // We need to handle cases where hispanic origin is other with or without write-in
-                else if (hispanicOrigin != null && hispanicOrigin.Length > 0 && DETHNIC1 == "N" && DETHNIC2 == "N" && DETHNIC3 == "N")
```

Comment:
The code seems functionally fine, for readability it may be a bit more readable if it were changed to the following pattern we use in a few other places.

```
string[] validHispanicOrigins = { "Cuban", "2182-4", "Puerto Rican", "2180-8", "Mexican", "2148-5" };

else if (hispanicOrigin != null && hispanicOrigin.Any(element => validHispanicOrigins.Contains(element)))
```